src/Sulu/Bundle/MediaBundle/Media/ImageConverter/ImagineImageConverter.php

```diff
@@ -401,15 +402,20 @@ private function modifyAllLayers(ImageInterface $image, callable $modifier)
 
         if (count($layers)) {
             $countLayer = 0;
-            $image->layers()->coalesce();
+
+            // coalesce currently not supported by the VipsAdapter:
+            // see: https://github.com/rokka-io/imagine-vips/issues/
```

Comment:
Issue number would also be nice :sweat_smile:

Reply:
@wachterjohannes https://github.com/rokka-io/imagine-vips/issues/8